extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -1502,26 +1504,37 @@ export class AmpStoryPage extends AMP.BaseElement {
    * Has to be called directly before playing the video.
    * @private
    */
-  startMeasuringVideoPerformance_() {
+  startMeasuringAllVideoPerformance_() {
     if (!this.mediaPerformanceMetricsService_.isPerformanceTrackingOn()) {
```

Comment:
can we get rid of this check since we're already doing it at the `startMeasuringVideoPerformance_` method?

Reply:
Both are used by different code paths